src/nimble.nim

```diff
@@ -858,6 +860,22 @@ js   - Compile using JavaScript backend.""",
     pkgRoot
   )
 
+  # Add vcs to the new nimble project
+  let vcsBin = options.action.vcsOption
+  if findExe(vcsBin, true).len != 0:
```

Comment:
This should be checked at the beginning of the function, if it doesn't exist then you should throw a NimbleError.

The actual logic you have here should still happen at the end though.

Reply:
ok, i  put the check code to the beginning of the init function and the actual logic is still at the end of the init function